snmp/datadog_checks/snmp/data/profiles/fortinet-fortigate.yaml

```
@@ -0,0 +1,232 @@
+      name: fgSysMemUsage
+  - MIB: FORTINET-FORTIGATE-MIB
+    symbol:
+      # Total physical memory (RAM) installed (KB).
+      OID: 1.3.6.1.4.1.12356.101.4.1.5
+      name: fgSysMemCapacity
+  - MIB: FORTINET-FORTIGATE-MIB
+    symbol:
+      # Current lowmem utilization (percentage). Lowmem is memory available for the kernel's own data structures and kernel specific tables. The system can get into a bad state if it runs out of lowmem.
+      OID: 1.3.6.1.4.1.12356.101.4.1.9
+      name: fgSysLowMemUsage
+  - MIB: FORTINET-FORTIGATE-MIB
+    symbol:
+      # Total lowmem capacity (KB). See fgSysLowMemUsage for the description of lowmem.
+      OID: 1.3.6.1.4.1.12356.101.4.1.10
+      name: fgSysLowMemCapacity
+
+  ### Disk
+  - MIB: FORTINET-FORTIGATE-MIB
+    symbol:
+      # Current hard disk usage (MB), if disk is present.
+      OID: 1.3.6.1.4.1.12356.101.4.1.6
+      name: fgSysDiskUsage
+  - MIB: FORTINET-FORTIGATE-MIB
+    symbol:
+      # Total hard disk capacity (MB), if disk is present.
+      OID: 1.3.6.1.4.1.12356.101.4.1.7
+      name: fgSysDiskCapacity
+
+  ### Virtual Domains
+  # A table of virtual domains configured on the device.
+  - MIB: FORTINET-FORTIGATE-MIB
+    table:
+      OID: 1.3.6.1.4.1.12356.101.3.2.1
+      name: fgVdTable
+    symbols:
+      # Operation mode of the virtual domain: nat(1), transparent(2)
+      - OID: 1.3.6.1.4.1.12356.101.3.2.1.1.3
+        name: fgVdEntOpMode
+      # HA cluster member state of the virtual domain on this device: master(1), backup(2), standalone(3)
+      - OID: 1.3.6.1.4.1.12356.101.3.2.1.1.4
+        name: fgVdEntHaState
+      # CPU usage of the virtual domain (percentage).
+      - OID: 1.3.6.1.4.1.12356.101.3.2.1.1.5
+        name: fgVdEntCpuUsage
+      # Memory usage of the virtual domain (percentage).
+      - OID: 1.3.6.1.4.1.12356.101.3.2.1.1.6
+        name: fgVdEntMemUsage
+      # Number of active sessions on the virtual domain.
+      - OID: 1.3.6.1.4.1.12356.101.3.2.1.1.7
+        name: fgVdEntSesCount
+      # The session setup rate on the virtual domain.
+      - OID: 1.3.6.1.4.1.12356.101.3.2.1.1.8
+        name: fgVdEntSesRate
+    metric_tags:
+      - tag: virtualdomain_index
+        column:
+          OID: 1.3.6.1.4.1.12356.101.3.2.1.1.1
+          name: fgVdEntIndex
+      - tag: virtualdomain_name
+        column:
+          OID: 1.3.6.1.4.1.12356.101.3.2.1.1.2
+          name: fgVdEntName
+
+  - MIB: FORTINET-FORTIGATE-MIB
+    symbol:
+      # The number of virtual domains in vdTable.
+      OID: 1.3.6.1.4.1.12356.101.3.1.1
+      name: fgVdNumber
+  - MIB: FORTINET-FORTIGATE-MIB
+    symbol:
+      # The maximum number of virtual domains allowed on the device as allowed by hardware and/or licensing.
+      OID: 1.3.6.1.4.1.12356.101.3.1.2
+      name: fgVdMaxVdoms
+
+  ### Interfaces
+  # Fortinet specific extensions to MIB-2 ifTable.
+  - MIB: FORTINET-FORTIGATE-MIB
+    table:
+      OID: 1.3.6.1.4.1.12356.101.7.2.1
+      name: fgIntftable
```

Typo (although not impactful since this field is mostly descriptive)

```suggestion
      name: fgIntfTable
```